kubernetes/client/ws_client_test.py

```diff
@@ -32,6 +36,24 @@ def test_websocket_client(self):
                 ]:
             self.assertEqual(get_websocket_url(url), ws_url)
 
+    @patch.object(websocket._http, '_can_use_sni')
+    @patch.object(websocket._http, '_open_socket')
+    @patch.object(ssl.SSLContext, 'load_verify_locations')
+    def test_use_of_correct_ca_cert(self, load_verify_locations, *ignored):
+        # arrange
+        expected = '/expected-ca.pem'
+        configuration.ssl_ca_cert = expected
+        load_verify_locations.side_effect = Exception("need not go further")
+
+        # act
+        with self.assertRaises(Exception):
```

Comment:
Why you need to raise and error here?

If you do not return an exception as side_effect, but return something that the code will understand. You can remove the assertRaises